Homework/homework4/README.md

```diff
@@ -0,0 +1,11 @@
+# Instructions: How to run a doctest
+
+1. Install Python 3.8 (https://www.python.org/downloads/)
+2. Install pytest `pip install pytest`
+3. Clone the repository https://github.com/ValeriL/Epam_Python \
+    `git clone https://github.com/ValeriL/Epam_Python`
+4. Checkout branch hw4_4 \
+    `git checkout hw4_4`
+6. Run a doctest for test_task4.py \
+    `pytest Homework/homework4/test_task4.py`
```

Comment:
Pytest does not run doctest by default.  Why does it work for you?

Reply:
So the point is to make pytest see doctests in this file and check it without straight mention "test" in the name of the file?

Reply:
Create a folder outside this repository. Copy your file there and run pytest.